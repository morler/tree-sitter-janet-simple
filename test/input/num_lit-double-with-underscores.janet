2_3_1__._1_2_e-1