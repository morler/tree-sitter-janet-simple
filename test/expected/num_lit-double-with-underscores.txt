(source [0, 0] - [0, 16]
  (num_lit [0, 0] - [0, 16]))
